test/unit/alert_test.rb

```diff
@@ -15,9 +15,8 @@ def test_by_level
     FactoryBot.create(:limit_alert, level: 50)
     FactoryBot.create(:limit_alert, level: 80)
 
-    alerts = Alert.by_level(80)
-    assert_kind_of Alert::ActiveRecord_Relation, alerts
-    assert_equal 1, alerts.count
+    alert = Alert.by_level(80).last
```

Comment:
What about

```
assert_equal [alert], Alert.by_level(80).to_a
```

Reply:
Done

Reply:
well not really what I expected :) you need to modify the line 16 into

```
alert = FactoryBot.create(:limit_alert, level: 80)
```